### PR TITLE
A few Requsitions changes

### PR DESCRIPTION
**Pull request by Metroidlover**
## About The Pull Request

So, a few changes on request of MJP
1: There is now a crate that you can purchase for two fully stocked non-captain Mateba belts
2: You can now buy Heavy/PW-MX revolver speedloaders and Incinidary pistol mags
3: All revolver crates now contain 5 more speedloaders because revolvers burn ammo as fast as I burn through anime songs to keep my sanity high enough to work
4: All non-spec related ammo crates now cost ten points less to be reasonable
5: The Captain's Mateba now uses the basic Mateba sprite and vice-versa

## Why It's Good For The Game
From MJP:
"Makes the Mateba appear in req platform so people can actually use it sometimes other than in EORD! It's pricy and order only though!

Also adds in special pistol ammo to order as well as making standard ammo more common and less pricy in req terminal across the board.

Also changes the captain Mateba sprite to the standard Mateba one since it looks nicer."

## Changelog
:cl: MJP & MetroidLover
add: There is now a crate that you can purchase for two fully stocked non-captain Mateba belts
add: You can now buy Heavy/PW-MX revolver speedloaders and Incinidary pistol mags
tweak: The Captain's Mateba now uses the basic Mateba sprite and vice-versa
balance: All revolver crates now contain 5 more speedloaders
balance: All non-spec related ammo crates now cost ten points less
/:cl:

### CODE REVIEW

**Comment by Metroidlover** [comment]
FUCKING NUMPAD ENTER KEY

**Comment by Rohesie** [comment]
PW-MX is super powerful. If people catch up to its potential they can delete things in short time, at short distances.

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -1566,7 +1583,7 @@ AMMO
 					/obj/item/ammo_magazine/rifle/ap,
 					/obj/item/ammo_magazine/rifle/ap
 					)
-	cost = RO_PRICE_NORMAL
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]: This one shouldn't be cheaper.

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -1583,11 +1600,76 @@ AMMO
...
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary,
+					/obj/item/ammo_magazine/pistol/incendiary
+					)
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_NORMAL
```

File: code/datums/supplypacks.dm [hunk 3]
```diff
@@ -1583,11 +1600,76 @@ AMMO
...
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy,
+					/obj/item/ammo_magazine/revolver/heavy
+					)
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_NORMAL
```

File: code/datums/supplypacks.dm [hunk 4]
```diff
@@ -1583,11 +1600,76 @@ AMMO
...
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba,
+					/obj/item/ammo_magazine/revolver/mateba
+					)
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_NORMAL
```

File: code/datums/supplypacks.dm [hunk 5]
```diff
@@ -1773,7 +1855,7 @@ AMMO
 	contains = list(
 					/obj/item/big_ammo_box/ap
 					)
-	cost = RO_PRICE_NORMAL
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_NORMAL
```

File: code/datums/supplypacks.dm [hunk 6]
```diff
@@ -1857,7 +1939,7 @@ AMMO
 					/obj/item/smartgun_powerpack,
 					/obj/item/smartgun_powerpack
 					)
-	cost = RO_PRICE_CHEAP
+	cost = RO_PRICE_VERY_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_CHEAP
```

File: code/datums/supplypacks.dm [hunk 7]
```diff
@@ -1883,7 +1965,7 @@ AMMO
 					/obj/item/ammo_magazine/flamer_tank,
 					/obj/item/ammo_magazine/flamer_tank
 					)
-	cost = RO_PRICE_NORMAL
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_NORMAL
```

File: code/datums/supplypacks.dm [hunk 8]
```diff
@@ -1999,7 +2081,7 @@ AMMO
 					/obj/item/ammo_magazine/minisentry,
 					/obj/item/ammo_magazine/minisentry
 					)
-	cost = RO_PRICE_NORMAL
+	cost = RO_PRICE_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_NORMAL
```

File: code/datums/supplypacks.dm [hunk 9]
```diff
@@ -2020,7 +2102,7 @@ AMMO
 	contains = list(
 					/obj/machinery/vending/lasgun
 					)
-	cost = RO_PRICE_PRICY
+	cost = RO_PRICE_KINDA_PRICY
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_PRICY
```

File: code/datums/supplypacks.dm [hunk 10]
```diff
@@ -2060,7 +2142,7 @@ AMMO
 					/obj/item/cell/lasgun/M43/highcap,
 					/obj/item/cell/lasgun/M43/highcap
 					)
-	cost = RO_PRICE_CHEAP
+	cost = RO_PRICE_VERY_CHEAP
```
Comment (Rohesie) [inline code/datums/supplypacks.dm]:
```suggestion
	cost = RO_PRICE_CHEAP
```

**Comment by Rohesie** [comment]
There are several unnecessary buffs here.

**Comment by Metroidlover** [comment]
Sorry to only get to this now, but do you want ALL of the ammo price changes reverted?

**Comment by Rohesie** [comment]
Not all, only those that are relevant balance-wise.
Stuff like ammo you can get in vendors in large numbers should be dirt cheap.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**